build.gradle.kts

```diff
@@ -137,9 +138,45 @@ subprojects {
 }
 
 // == Kover setup ==
-subprojects {
-    if (project.name in uncoveredProjects) return@subprojects
-    apply(plugin = "kover-conventions")
+kover {
+    if (hasProperty("kover.enabled") && property("kover.enabled") != "true") {
+        disable()
+    }
+
+    currentProject {
```

Comment:
Can we extract Kover configuration to a separate file?

Reply:
Sure